### PR TITLE
[SB-6] Add basic authentification using Devise gem

### PR DESCRIPTION
**Pull request by m3xq**
**Link to the ticket**
https://trello.com/c/0lnF9Pho/13-sb-6-add-basic-authentification-using-devise-gem

**Description**
Add basic devise functions and views

**After**

1. Login

login

2. Registration

reg

3. Errors

errors

4. Notice

no

### CODE REVIEW

**Comment by m3xq** [comment]
[[SB-6] Add basic authentification using Devise gem](https://trello.com/c/0lnF9Pho/13-sb-6-add-basic-authentification-using-devise-gem)

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                    </div>
+
+                    <div class="actions mt-4 text-center">
+                        <%= f.submit "Change", class: "btn btn-primary" %>
+                    </div>
+                <% end %>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (B0RlS) [inline app/views/devise/passwords/edit.html.erb:37]:
```suggestion
</div>

```

File: spec/system/log_ins_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'LogIns', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
```
Comment (B0RlS) [inline spec/system/log_ins_spec.rb:6]:
```suggestion
  before { driven_by(:rack_test) }
```

File: spec/system/log_ins_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  let(:user) { FactoryBot.create(:user) }
```
Comment (B0RlS) [inline spec/system/log_ins_spec.rb:8]:
```suggestion
  let(:user) { create(:user) }
```
Check if it works like this

File: spec/system/registrations_spec.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Registrations', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
```
Comment (B0RlS) [inline spec/system/registrations_spec.rb:6]:
```suggestion
  before { driven_by(:rack_test) }
```

**Pull request merged**